portfolio/src/main/webapp/index.html

```diff
@@ -2,13 +2,10 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio - Home</title>
     <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
-    <link href='https://fonts.googleapis.com/css?family=Sora' rel='stylesheet'>
-    <link href='https://fonts.googleapis.com/css?family=Be Vietnam' rel='stylesheet'>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
How will you get the font-awesome styles then? I see that you included some in `style.css` but not for `fa-instagram` and other icons for example.

Reply:
oops, I just realised that the icons disappeared because of this